webpack/assets/javascripts/react_app/components/Layout/components/__snapshots__/NavDropdown.test.js.snap

```diff
@@ -1,7 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`NavDropdown rendering render NavDropdown 1`] = `
-<ForwardRef
+<Uncontrolled(Dropdown)
```

Comment:
is this intentional?

Reply:
guess not since the tests are failing on this

Reply:
no, not really ... just blind `-u` :(